docs/source/concepts/functionality/interfaces/module_based/output_formats.rst

```diff
@@ -1,38 +1,178 @@
-.. dropdown:: Distribution Statement
+Using GeoIPS Output Checkers Tutorial
```

Comment:
Is the output formatter docs replaced with output checkers here? Was that intended?

Reply:
Fixed.